include/zephyr/bluetooth/audio/has.h

```diff
@@ -67,6 +67,26 @@ enum bt_has_capabilities {
 	BT_HAS_PRESET_SUPPORT = BIT(0),
 };
 
+/* Preset event types */
+enum bt_has_preset_event_type {
+	BT_HAS_PRESET_DELETE,
+	BT_HAS_PRESET_AVAILABLE,
+	BT_HAS_PRESET_UNAVAILABLE,
+	BT_HAS_PRESET_RECORD_UPDATE,
+	BT_HAS_PRESET_RECORD_READ_RSP,
+};
+
+/* Preset record event parameter */
+struct bt_has_preset_record {
+	enum bt_has_properties properties;
+	char name[];
+};
+
+/* Preset delete event parameter */
+struct bt_has_preset_delete {
+	uint8_t end_index;
+};
```

Comment:
Wouldn't mind if the fields in these structs had doxygen comments/documentation :)